File: src/main/resources/static/js/socket/Message.js
class Message {
	constructor(contents, playerId) {
		this.contents = contents;
		this.playerId = playerId;
	}

	getPayload() {
		const payload = {};
		if (this.contents) payload.contents = this.contents;
		if (this.playerId) payload.playerId = this.playerId;
		return payload;
	}

}

File: src/main/java/com/marcosalles/bolgame/model/Message.java
package com.marcosalles.bolgame.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Message<T extends Object> implements Serializable {

	private String playerId;
	private T contents;
}
